src/v2/cookbook/unit-testing-vue-components.md

```diff
@@ -155,32 +158,34 @@ describe('Foo', () => {
     }
   })
 
-  // see if the message renders
+  // see if the `message` renders
```

Comment:
I'm not sure I'd wrap this in backticks, because it's not necessarily referring to the message class, it's referring to the message itself. (i.e. the content)